debian/changelog

```diff
@@ -1,3 +1,10 @@
+aeneas (1.7.3.1) stable; urgency=medium
+
+  * Fix spurious warnings about "encoding is not UTF-8" when encoding is "utf-8"
+  * Add debian packaging for python3-aeneas
+
+ -- Stephen McConnel <stephen_mcconnel@sil.org>  Thu, 02 Apr 2020 16:45:00 -0600
+
```

Comment:
@StephenMcConnel, Drat, I changed this while testing and forgot to include here:

`E: python3-aeneas: debian-changelog-file-contains-invalid-email-address alberto@nievo`

So `alberto@nievo` should be changed to `alberto@readbeyond.it` 

I got that address from https://www.albertopettarin.it/contact.html

Reply:
I added the line to the lintian-overrides files.